.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+
+#### Description
+
+**Summary of the change**: Provide a concise summary of what this PR is about. Describe the changes made in a single sentence or short paragraph.
+
+**Detailed description**:
+- **What**: Detail what changes have been made in the PR.
+- **Why**: Explain the reasons behind the changes. Why were they necessary?
+- **How**: Describe how the changes were implemented, including any key aspects of the code modified or new features added.
+
+---
+
+#### Type of Change
+
+Please mark the relevant options with an "X":
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Documentation update
+
+Updates/remove existing public API methods:
+- [ ] Is breaking change
+
+
+#### Checklist
+
+- [ ] Unit tests are in place
+- [ ] The build is green
+- [ ] All public methods affected by my PR has up to date API docs
```

Comment:
I considered suggesting an extra item to confirm that the Detekt task is also successful, but I believe this is covered by "The build is green".

Reply:
Fixed